tests/Doctrine/Tests/Sniffs/Classes/ExceptionInterfaceNamingSniffTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Doctrine\Tests\Sniffs\Classes;
+
+use Doctrine\Sniffs\Classes\ExceptionInterfaceNamingSniff;
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Files\LocalFile;
+use PHP_CodeSniffer\Ruleset;
+use PHPUnit\Framework\TestCase;
+
+class ExceptionInterfaceNamingSniffTest extends TestCase
```

Comment:
Please use `SlevomatCodingStandard\Sniffs\TestCase` as a base class and then its provided testing helpers, just like tests in Slevomat CS itself: https://github.com/slevomat/coding-standard/blob/master/tests/Sniffs/Types/EmptyLinesAroundTypeBracesSniffTest.php